src/main.cpp

```diff
@@ -829,7 +831,10 @@ bool ContextualCheckTransaction(
     assert(!saplingActive || overwinterActive); // Sapling cannot be active unless Overwinter is
     assert(!heartwoodActive || saplingActive);  // Heartwood cannot be active unless Sapling is
     assert(!canopyActive || heartwoodActive);   // Canopy cannot be active unless Heartwood is
-    assert(!futureActive || canopyActive);      // ZFUTURE must always include the latest live version
+    assert(!nu5Active || canopyActive);         // Nu5 cannot be active unless Canopy is
+    assert(!futureActive || nu5Active);         // ZFUTURE must always include the latest live version
```

Comment:
The comment isn't quite accurate. It enforces that ZFUTURE includes NU5, but NU5 isn't live yet.